SolidStateGroup/bullet-train-api
# feat: Open payment modal if a plan was preselected, add annual plans

Thanks for submitting a PR! Please check the boxes below:


## Changes

**The annual plans need adjusting in Chargebee before this goes live**

Navigating to app.flagsmith.com from the [pricing page](https://www.flagsmith.com/pricing), or simply navigating to https://app.flagsmith.com/signup?plan=startup-v2 will cookie the plan. Also adds a toggle for annual plans.

Upon hitting the features page for the first time it will take that plan and open checkout if payments are enabled.

## How did you test this code?

- Go to /signup?plan=startup-v2 on the preview url
- Signup, create org, create project
- See payment modal

![image](https://pouch.jumpshare.com/preview/HVgK_5PMfVgDMFViW-PTXj-ZPLQba7Q0PJSfvY2MmT1H91Sw4eB3lgq74pFiVYqGZwp_coD_77DQ8C9ijCgShkQ9b_Q_CAUcloaeCnr-T40)

Attempt to purchase an annual plan

![image](https://pouch.jumpshare.com/preview/2r4Qzfl2_T_mR71mxbyI1BsQZ1JIjBfKMEvM85PkhENeSiL0Fq9w7ASESlz8NerxZwp_coD_77DQ8C9ijCgShiGufCFwBo4j7iYUQfTyaCI)
